### ISSUE TITLE
[Images App] Image not displayed in controller in Firefox

### ISSUE THREAD

**Issue by jamesscottbrown** [ove/ove-apps#195]
If the control page for the Images App is opened in Chrome, rather than Firefox, then the preview image is not displayed; this makes it difficult to acurately zoom or pan.

**Comment by senakafdo** [comment]
@jamesscottbrown, I am unable to reproduce this issue. I have loaded the Images App using the launcher and selected the HighSmith state. I have tried this in both Chrome and Firefox and I have been able to see the controller launch successfully and also make use of it.

I'm guessing that there must have been some issue preventing the display of the image. Perhaps you saw a cached version of the image on one browser and the other browser was unable to display it?

I'm resolving this issue for now as Invalid, but please reopen if you can reproduce this issue consistently and include the steps to reproduce the issue when you do so.

**Comment by jamesscottbrown** [comment]
I've posted the URL of an example image (hosted by the Asset Manager) that reproduces the problem to our Microsoft Teams channel.

The problem only seems to occur for some files - possibly those whose size exceeds some threshold.

**Comment by senakafdo** [comment]
@jamesscottbrown, thanks. Lets also host the same image outside of the Asset Manager and see if it creates the same issue.

**Comment by senakafdo** [comment]
I have tested using the provided image on both Chrome and Firefox, and it seems that it works well on Chrome and not on Firefox. I have hosted the image outside of AM, and the results are the same. 
I'm unsure of what the reason is, but I also observed that Firefox takes longer to load the image compared to Chrome, and there may be some connection. I don't expect this to be an OVE (or even an OpenSeadragon) bug but potentially a limitation of the web browser since the same works well when tested with Safari.
